www_src/lib/api.js

```diff
@@ -54,11 +54,11 @@ module.exports = function (options, callback) {
 
     // Set cache if window.Android is available
     if (window.Android) {
-      window.Android.setSharedPreferences(key, JSON.stringify(body), false);
+      window.Android.setMemStorage(key, JSON.stringify(body), false);
     }
 
     // If there is a callback, forward the response body
-    if(callback) {
+    if (callback) {
       callback(null, body);
```

Comment:
nit: there is no good reason to ever use `null` in JS context. Can we make this "false" if we're editing this file anyway?